recipes/haploclique/1.3.1/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "haploclique" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "2ea8c6089d3f971c0fd5d20e708905521e1b6f802a8835d2cd2817f48e442a9c" %}
+
+about:
+    summary: Viral quasispecies assembly via maximal clique finding. A method to reconstruct viral haplotypes and detect large insertions and deletions from NGS data.
+    home: https://github.com/cbg-ethz/{{ name }}
+    license: GPLv3
```

Comment:
Can we add lincese_file: as well please. GPL want to have a copy of the license distributed with the software.

Reply:
Okay, I'll ask @armintoepfer upstream.

Reply:
@DrYak please refer to your PI about the license.

Reply:
Isn't the code already licensed under "[*GNU General Public License* {...} *version 3* {...} *or* {...} *any later version*](https://github.com/cbg-ethz/haploclique/blob/7707f1ab143f2c2b4b7b3b01bc51f86d53ac710e/src/haploclique.cpp#L6)" ?

Or shall I re-check with the Professor ?

Reply:
In that case, please make a PR for the license

Reply:
I've [just done it](https://github.com/cbg-ethz/haploclique/pull/61)

Reply:
Merged